frontend/src/scenes/feedback/Feedback.tsx

```diff
@@ -271,13 +24,26 @@ export const Feedback = (): JSX.Element => {
                         </LemonTag>
                     </div>
                 }
-                caption={<p>Hear what your users have to say about your product.</p>}
             />
-            <FeedbackWidgetTab
-                config={{
-                    eventName: 'Feedback Sent',
-                    feedbackProperty: '$feedback',
+            <LemonTabs
+                activeKey={activeTab}
+                onChange={(key) => {
+                    setActiveTab(key)
                 }}
+                tabs={[
+                    {
+                        content: <InAppFeedback />,
+                        key: 'in-app-feedback',
+                        label: 'In-app feedback',
+                        tooltip: 'Analyze feedback from your users',
+                    },
+                    {
+                        content: <UserInterviewScheduler />,
+                        key: 'user-interview-scheduler',
+                        label: 'User interview scheduler',
+                        tooltip: 'Schedule user interviews with your users',
+                    },
```

Comment:
These tooltips don't seem very useful. There's always a cost in terms of mental load when adding an info icon + tooltip, so we should only add them when they can really clarify something. It does not make things particularly clearer to say that an interview scheduler allows scheduling interview.